master/internal/rm/kubernetesrm/pods.go

```diff
@@ -202,7 +204,11 @@ func Initialize(
 
 	err = p.startNodeInformer()
 	if err != nil {
-		panic(err)
+		if strings.Contains(err.Error(), "nodes is forbidden") {
```

Comment:
is there a smarter way to if `err` results from a permissions issue besides string matching?

Reply:
Also is `nodes` present in every case? Or is that something that varies between different configurations?